_includes/languages.html

```diff
@@ -0,0 +1,50 @@
+<script type="text/javascript" src="{{site.baseurl}}/assets/js/jquery-3.2.1.min.js"></script>
+<script type="text/javascript" src="{{site.baseurl}}/assets/js/stellarnav.min.js"></script>
+<script type="text/javascript" src="{{site.baseurl}}/assets/js/materialize.min.js"></script>
+
+<script src="{{site.baseurl}}/assets/js/init.js"></script>
+{% include lightbox.html %}
+</body>
+
+
```

Comment:
@ashiscs should we remove these lines?

Reply:
Yes sir @leecalcote this needs to be removed!!